Resources/Audio/_DV/Lobby/attributions.yml

```diff
@@ -0,0 +1,20 @@
+# Ordered by filename alphabetically
+- files: ["all_grown_up.ogg"]
```

Comment:
theyre all the same so have them all in 1 entry
if it doesn't exist anywhere else on the internet just say its created for Delta-V and link the repo instead of a bunch of google drive links